src/static/styles/main.scss

```diff
@@ -263,6 +263,7 @@ h6,
 
 .device-data-piechart {
   max-height: 20em;
+  height: 100%;
```

Comment:
Now it shows up in Chrome. But when you have a long list of interfaces, the graph drifts away from its legend/labels:

![Screenshot_20201014_184609](https://user-images.githubusercontent.com/60540759/96020008-b234af80-0e4d-11eb-84ae-fc640c26fd4b.png)